src/styles/blocks/catalog.scss

```
@@ -18,8 +20,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $during-anim;
```

it's better to explicitly specify what properties you put on
```suggestion
    transition: 
      color $during-anim,
      scale $during-anim;
```